02-analytics/technical.md

```diff
@@ -152,33 +155,6 @@ Use `/track` endpoint. Docs: https://docs.rudderstack.com/rudderstack-api/api-sp
 | Element Type     | elementType     | episode, series, ...                                                                       |
 | Element ID       | elementId       | id of the clicked element                                                                  |
 
-## Section rendered (video_slider_rendered)
-
-*When*: When items in horizontal collection that contains videos are rendered for the first time. This happens when a user visits the home page for the first time or scrolls down on said page. Featured corousel is not included (for now).
-
-*Reason*: This helps us see how what content the user has looked at, e.g. on the home page.
-
-### API
-
-Use `/track` endpoint. Docs: https://docs.rudderstack.com/rudderstack-api/api-specification/rudderstack-spec/track
-
-### Data
-
-| Data                 | Name               | Comments                                         |
-|----------------------|--------------------|--------------------------------------------------|
-| Event ID             | event              | Hardcoded: `video_slider_rendered`               |
-| Section ID       | sectionId       | ID of the section                             |
-| Section Name     | sectionName     | For easier identification in tools            |
-| Section Position | sectionPosition | int, position in the page's list of sections  |
-| Section Type     | sectionType     | slider, featured, etc                         |
-| Page Name        | pageName        | same ID as for page/screen tracking           |
-
-### Page/Screen IDs
-
-This are the currently used screens from the app:
-
-* ExplorePage
-
 ## Section click (section_clicked)
```

Comment:
**sectionType**: Kan vi bruke __typename fra graphql? DefaultSection, DefaultGridSection etc etc etc.
**elementType**: Kan vi bruke item.__typename fra graphql? Episode, Show, etc etc.

Reply:
Episode
```json
    "properties": {
        "elementId": "1618",
        "elementPosition": 2,
        "elementType": "Episode",
        "sectionId": "1",
        "sectionName": "Latest episodes",
        "sectionPosition": 5,
        "sectionType": "DefaultSection"
    },
```

Show
```json
    "properties": {
        "elementId": "93",
        "elementPosition": 0,
        "elementType": "Show",
        "sectionId": "55",
        "sectionName": "Våre høytidsstevner",
        "sectionPosition": 11,
        "sectionType": "DefaultSection"
    },
```

Reply:
I think that sounds sane.